src/Console/TinkerCommand.php

```diff
@@ -53,6 +54,11 @@ public function fire()
         $shell->addCommands($this->getCommands());
         $shell->setIncludes($this->argument('include'));
 
+        (new ClassAliasAutoloader(
+            $shell, 
+            $this->getLaravel()->basePath().'/vendor/composer/autoload_classmap.php'
```

Comment:
@GrahamCampbell I had been using `base_path()` but I assume that helper shouldn't be relied upon here. This seem like a Laravel/Lumen compatible way to find the class map?